src/doc/gui_manual/Operators/OperatorTypes/PICS/Poincare_operator.rst

```diff
@@ -0,0 +1,318 @@
+    toroidal winding to have a set value. Zero indicates no override. 
+
+Override poloidal winding
+    In some cases such as debugging, it may be informative to force the
+    poloidal winding to have set value. Zero indicates no override.
+
+Winding pair confidence (Range 0-1, Default 0.9)
+    Sets the limit for the number of mismatches in the consistency in the
+    winding pairs.
+
+Detect Rational Surface
+    Allows for the construction of rational surfaces via an iterative process.
+
+    * Maximums iterations – sets the number of iterations for constructing the
+    rational surface. Typically, it can be found with 5-10 iterations. 
+
+Note: The rational surface construction is experimental code and does not
+always work!!!! 
+
+Detect O Points
+    Allows for the detection of O points in “island chains” via an iterative
+    process.
+    
+    * Maximum iterations - sets the number of iterations for locating the O
+      point. Typically, it can be found with 5 iterations. 
+
+Note: The critical point detection is experimental code and does not always
+work!!!! 
+
+
+Perform O-Line Analysis
+    Calculate the poloidal winding relative the O-Line (central axis) which
+    provides a more accurate winding value.
+
+    * O-Line toroidal windings (Default 1) – sets the toroidal winding value,
+      i.e. the period (for the central axis the period would be 1). 
+    * O-Line Axis Point File - allows the user to select a text file containing
+      the points along the axis from 0 to 360 degrees (note there is no overlap
+      P(0) != P(n)).
+
+Show chaotic fieldlines as points
+    Because chaotic fieldlines can not be classified, they are not displayed
+    unless this is checked. 
+
+Show islands only
+    Culls the results so that only island chains are displayed. 
+
+Show ridgelines
+    Displays the 1D plots of the distance and ridgeline samples.
+
+Verbose
+    Dumps information regarding the analysis to the terminal. The final summary
+    may be useful. For example: :: 
+   
+   Surface id = 0 < 2.35019 0 0.664124 > 121:11 121:11 (11) flux surface with 4
+     nodes (Complete) 
+   Surface id = 0 
+   seed location < 2.35019 0 0.664124 > 
+   the winding pair 121:11 
+   the toroidal:poloidal periods (as a winding pair) 121:11 
+   the multiplication faction (11) i.e. diving by this number will give the
+     base winding values, in this case 11:1. 
+   surface type: flux surface 
+   number of nodes in each winding group: with 4 nodes 
+   analysis state: complete. 
+
+
+Appearance
+^^^^^^^^^^
+
+The appearance tab specifies how the integral curve will be rendered. In addition
+to the Appearance attributes common to all PICS operators, the Poincare
+operator supports the following attributes.
+
+
+Data Value
+""""""""""
+
+Allows the user to set the data value associated with each data point and the
+overall limits
+
+None
+    Solid color from the single color 
+
+Safety Factor Q
+    Use the safety factor 
```

Comment:
Use it to do what...effect the coloring?

Reply:
Yes, it is the coloring. I do not know yet what P and Q are talking about here, but I've made a note to investigate it.